internal/ethapi/api.go

```diff
@@ -510,12 +511,13 @@ func (s *PrivateAccountAPI) Unpair(ctx context.Context, url string, pin string)
 // PublicBlockChainAPI provides an API to access the Ethereum blockchain.
 // It offers only methods that operate on public data that is freely available to anyone.
 type PublicBlockChainAPI struct {
-	b Backend
+	b    Backend
```

Comment:
What do you think about adding a method `DiffDb` of `GetDiff` to the `Backend` interface instead of passing the diffdb through multiple layers of APIs?

The `Backend` has an `Ethereum` struct attached to it, and the `Ethereum` struct has a diffdb
https://github.com/ethereum-optimism/go-ethereum/blob/931ec90d434d04d2ec7e45b21d8e07e0cb23fb2c/eth/api_backend.go#L43-L45